libmachine/libmachine.go

```diff
@@ -31,7 +31,6 @@ type API interface {
 	NewHost(driverName string, rawDriver []byte) (*host.Host, error)
 	Create(h *host.Host) error
 	persist.Store
-	GetMachinesDir() string
```

Comment:
nevermind - this was moved into persist.Store so it's implicitly here too, which is why this is removed.